package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-config-habitrpg",
-  "version": "1.1.1",
+  "version": "1.2.1",
```

Comment:
We should use `npm version minor` to bump this after it's merged in so we get the correct tag notation in git.